cmd/internal/cli/action_flags.go

```diff
@@ -812,6 +813,15 @@ var actionOCIFlag = cmdline.Flag{
 	EnvKeys:      []string{"OCI"},
 }
 
+// -B|--bind
+var actionDevice = cmdline.Flag{
+	ID:           "actionDevice",
+	Value:        &device,
+	DefaultValue: []string{},
+	Name:         "device",
+	Usage:        "a CDI device mapping. A device mapping consists of a kind specification followed by the equal sign (=) and a device label (e.g. vendor.com/device=mydevice). Kind specifications consist of a prefix and a name, separated by a slash (e.g. vendor.com/device, in the previous example). Multiple device mappings can be given by a comma separated list.",
```

Comment:
I think the terms we use are `fully-qualified CDI device name` instead of `mapping`. The parts are:

`<VENDOR>/<CLASS>=<NAME>` and together these are a "fully-qualified CDI device name".